packages/components/src/components/alert/alert.scss

```diff
@@ -22,6 +23,16 @@
 
 $border-style: 1px solid var(--calcite-color-border-3);
 
+:host([scale="s"]) {
+  --calcite-internal-alert-action-spacing: var(--calcite-spacing-xxs);
+}
+:host([scale="m"]) {
+  --calcite-internal-alert-action-spacing: var(--calcite-spacing-xxs);
+}
+:host([scale="l"]) {
+  --calcite-internal-alert-action-spacing: var(--calcite-spacing-sm);
```

Comment:
These spacing values should be `-xs` for small and medium.

Reply:
nice catch!  Must have just read that wrong in Figma.  Too many x's!